source/server-discovery-and-monitoring/tests/README.rst

```diff
@@ -40,6 +42,13 @@ A response is a pair of values:
   The empty response `{}` indicates a network error
   when attempting to call "ismaster".
 
+An "applicationError" object has the following keys:
+
+- address: The source address, for example "a:27017".
+- generation: The error's generation number, for example `1`.
+- error: An error response, for example `{ok: 0, errmsg: "not master"}`.
+  The empty error response `{}` indicates a network error.
```

Comment:
I cargo culted the "`{}` indicates a network error" concept from "responses" above. Let me know if you have a better idea.

Reply:
I'd like to see something that allows us to represent a network timeout error.  So maybe we need a few extra sibling fields instead, e.g.

    networkError: true
    isTimeout: true